### ISSUE TITLE
Presence sensor ZY-M100

### ISSUE THREAD

**Issue by taston** [rospogrigio/localtuya#1544]
Does the localtuya support the ZY-M100 presence sensor (WiFi)? I tried to add, but there are no corresponding entities....

**Comment by juliovidalmoro** [comment]
yes... i do it... but i can't integrate light sense. I don't know why yet.

**Comment by taston** [comment]
Where i can find logs?
![1](https://github.com/rospogrigio/localtuya/assets/22216384/0aead982-8702-4bb9-8dff-f1ea233a89f5)
![2](https://github.com/rospogrigio/localtuya/assets/22216384/9afb3e76-7652-4e7e-aaeb-ccdae32e4db3)

**Comment by noodlenoggan** [comment]
Localtuya only provides the presence attribute. The other attributes are there but you need to manually add them. I'll see if I can get the matching id for each attribute.

**Comment by taston** [comment]
how can I add sensor (just only presence), I got error :-(

**Comment by noodlenoggan** [comment]
Does the regular tuya show the device in HA?

I have both tuya and localtuya integrations installed in HA. I separate them by adding -LT for localtuya so I know which is tuya and which is not when adding to the dashboard and automations.

**Comment by taston** [comment]
What i should set in this form? I want set only presence sensor 
![tuya](https://github.com/rospogrigio/localtuya/assets/22216384/a4f1f3bf-611a-4527-a9c1-337f926063d9)

**Comment by noodlenoggan** [comment]
You only need to select it. Leave the rest with their defaults.

**Comment by noodlenoggan** [comment]
I've switched to Tuya since LocalTuya is having connectivity issues.

https://github.com/rospogrigio/localtuya/issues/1530

Regular Tuya automatically gives you all but the lux sensor attribute in HA.

**Comment by taston** [comment]
> > 2023-10-30 12:49:41.914 WARNING (SyncWorker_8) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:42.010 WARNING (SyncWorker_9) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:42.609 WARNING (SyncWorker_4) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:42.955 WARNING (SyncWorker_1) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:43.143 WARNING (SyncWorker_3) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:43.650 WARNING (SyncWorker_8) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:43.938 WARNING (SyncWorker_9) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:44.120 WARNING (SyncWorker_4) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:44.615 WARNING (SyncWorker_1) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:44.888 WARNING (SyncWorker_3) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:46.912 WARNING (SyncWorker_8) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:48.949 WARNING (SyncWorker_9) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:49.934 WARNING (SyncWorker_4) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:50.923 WARNING (SyncWorker_2) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:53.910 WARNING (SyncWorker_5) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:54.899 WARNING (SyncWorker_10) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:55.887 WARNING (SyncWorker_0) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:56.956 WARNING (SyncWorker_6) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:57.785 WARNING (SyncWorker_2) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:57.858 WARNING (SyncWorker_8) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:58.320 WARNING (SyncWorker_7) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:58.830 WARNING (SyncWorker_10) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns
> 2023-10-30 12:49:59.342 WARNING (SyncWorker_6) [custom_components.localtuya.binary_sensor] [bf6...rba] State for entity binary_sensor.lazienka_tuya_local_obecnosc did not match state patterns

**Comment by noodlenoggan** [comment]
I would try using the regular Tuya integration to test the device first. It's working great for me.

Is your HA system updated?

**Comment by taston** [comment]
Tuya integration works good, but has little time delay. 

HA version 2023.10.3 / OS 11.0

**Comment by noodlenoggan** [comment]
Try updating HA. Below is my current version.

Home Assistant 2023.10.5
Supervisor 2023.10.1
Operating System 11.1
Frontend 20231005.0 - latest

**Comment by Theo16** [comment]
Tuya-local support this device:

https://raw.githubusercontent.com/make-all/tuya-local/main/custom_components/tuya_local/devices/zym100_presence_sensor.yaml
